models/professor.py

```diff
@@ -266,6 +268,12 @@ def get_departments_taught(self):
                 group(1).upper() for uc in ucs)
         return sorted(departments_taught)
 
+    def transfer_reviews_from(self, prof):
+        mcs = user_course.MenloCourse.objects(professor_id=prof.id)
```

Comment:
Also, you should remove the prof that you're transferring the reviews from so people can't accidentally add reviews for that prof anymore. Alternatively, you could have a default parameter delete_prof=False (that only if set to True will actually delete the prof), that acts as a reminder to the person using the method to consider if they need to remove the prof as well as transferring the reviews.